src/media-groups.js

```diff
@@ -522,12 +526,17 @@ export const initialize = {
 
         let properties = mediaGroups[type][groupId][variantLabel];
 
+        // TODO: we should filter for hls as well.
+        // Filter out unsupport playlists.
+        if (properties.playlists) {
+          properties.playlists = properties.playlists.filter((p) => !p.attributes || !p.attributes.CODECS);
```

Comment:
I think we should explicitly filter the im1t codec rather than anything that has codecs set because it's possible that someone has the vtt codec set (unlikely, but possible)